Pipfile

```diff
@@ -23,6 +23,8 @@ retrying = "==1.3.3"
 sqlalchemy = "==1.1.10"
 sqlalchemy-utils = "==0.32.14"
 structlog = "==17.2.0"
+requestsdefaulter = {git = "https://github.com/ONSdigital/requestsdefaulter.git", editable = true}
+flask-zipkin = "*"
```

Comment:
Does this not need pinning?

Reply:
Why? I can do but that's generally what the lockfile is for.

Reply:
https://github.com/ONSdigital/ras-party/pull/158 - Asked @benjefferies and @jcox-dev and it looks like this service just got missed when the other services were being changed